internal/maintenance/job_rescuer.go

```diff
@@ -232,11 +234,6 @@ func (s *JobRescuer) runOnce(ctx context.Context) (*rescuerRunOnceResult, error)
 			break
 		}
 
-		s.Logger.InfoContext(ctx, s.Name+": Rescued batch of jobs",
-			slog.Int64("num_jobs_discarded", res.NumJobsDiscarded),
-			slog.Int64("num_jobs_retried", res.NumJobsRetried),
-		)
```

Comment:
A couple of these services sort of had doubled up run lines. I took one half out and just left the line above.